libUtils/windowsChromeDriver.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import requests
+import os
+import zipfile
+
+class BrowserManager:
+    def __init__(self,extract_dir=".//dependencies//chrome-driver"):
+        self.extract_dir = extract_dir
+    def get_chrome_version(self):   #    Get the Chrome browser version installed on Windows.
+        try:
+            output = subprocess.check_output(
+                r'reg query "HKEY_CURRENT_USER\Software\Google\Chrome\BLBeacon" /v version',
+                shell=True,
+                encoding='utf-8'
+            )
+            # Extract version from the command output
+            version = output.split()[-1]
+            print(f"Chrome Version: {version}")
+            return version
+
+        except Exception as e:
+            raise Exception(f"Error getting Chrome version: {e}")
```

Comment:
reg_path  = HKEY_CURRENT_USER\Software\Google\Chrome\BLBeacon" /v version
Change the exception message to f"Failed to get chrome browser version. Error:  {e}"
Also add print statement before raise statement  f"Failed to read chrome browser version from path:  {reg_path  }"